ts/src/grpc.ts

```diff
@@ -231,7 +235,7 @@ export namespace grpc {
           const gRPCMessage = headers.get("grpc-message") || [];
           props.debug && debug("onHeaders.gRPCMessage", gRPCMessage);
           if (code !== Code.OK) {
-            rawOnError(code, gRPCMessage[0]);
+            rawOnError(code, gRPCMessage.length > 0 ? decodeURIComponent(gRPCMessage[0]) : undefined);
```

Comment:
`decodeURIComponent` can throw `URIError`, we should attempt to handle it. Could we also use the `""` (empty string) as the value rather than undefined?